astrobase/periodbase/kbls.py

```diff
@@ -1265,11 +1269,11 @@ def bls_stats_singleperiod(times, mags, errs, period,
 
         if magsarefluxes:
             blsmodel[transitindices] = (
-                blsmodel[transitindices] + thistransdepth
+                blsmodel[transitindices] - thistransdepth
             )
         else:
             blsmodel[transitindices] = (
-                blsmodel[transitindices] - thistransdepth
+                blsmodel[transitindices] + thistransdepth
```

Comment:
I think this should be left as a minus here because eebls.f definitely returns -ve transit depths for magnitudes. I made a plot of the snr['phases'] vs snr['phasedmags'] and snr['phases'] vs snr['blsmodel'] returned by this function currently and it seems to be doing the right thing:

![test](https://user-images.githubusercontent.com/1868184/45300241-d0e05380-b4db-11e8-93d7-909d82bd25d6.png)

This is with the test LC of an EB: HAT-772-0554686-V0-DR0-hatlc.sqlite.gz, which is in the bundle from the astrobase-notebooks repository.